src/main/java/core/basesyntax/figure/Figure.java

```diff
@@ -0,0 +1,27 @@
+package core.basesyntax.figure;
+
+public class Figure implements Draw {
+    private String color;
+    private String name;
+
+    public String getColor() {
+        return color;
+    }
+
+    public void setColor(String color) {
+        this.color = color;
+    }
+
+    public String getName() {
+        return name;
+    }
+
+    public void setName(String name) {
+        this.name = name;
+    }
+
+    @Override
+    public void toDraw() {
+
+    }
```

Comment:
do we need this?

Reply:
@MaliukDaria we need this to use toDraw() method from Figure class and by implementation of its interface, classes that extend Figure class, can inherit this interface too;